compass/landice/tests/dome/decomposition_test/__init__.py

```diff
@@ -26,12 +26,16 @@ def __init__(self, test_group, mesh_type):
         test_group : compass.landice.tests.dome.Dome
             The test group that this test case belongs to
 
+        velo_solver : str
```

Comment:
```suggestion
        velo_solver : {'sia', 'FO'}
```
I'm assuming there will only ever be these two, in which case listing them is a bit better than giving the type as `str`.